scripts/tests/traits/dual_wield.lua

```diff
@@ -0,0 +1,97 @@
+-----------------------------------
+-- Dual Wield Trait - Equipment TP Reset Prevention
+-- This test verifies that TP is preserved when equipment changes
+-- fail due to missing Dual Wield trait.
+--
+-- Note: Dual Wield is available on:
+--   - NIN (Ninja) - Level 10
+--   - DNC (Dancer) - Level 20
+--
+-- Fixes: https://github.com/HorizonFFXI/HorizonXI-Issues/issues/2268
+-----------------------------------
+
+describe('Dual Wield Trait - Equipment TP Reset Prevention', function()
+    ---@type CClientEntityPair
+    local player
+
+    before_each(function()
+        -- Spawn NIN/5 (before DW unlocks at level 10)
+        player = xi.test.world:spawnPlayer({
+            level = 5,
+            mjob = xi.job.NIN,
+            zone = xi.zone.WEST_RONFAURE
+        })
+    end)
+
+    it('should preserve TP when offhand equip fails without Dual Wield trait', function()
+        -- Arrange: Give player weapons and TP
+        player:addItem(xi.item.BRONZE_DAGGER)
+        player:addItem(xi.item.BRASS_DAGGER)
+
+        -- Equip main hand weapon
+        player:equipItem(xi.item.BRONZE_DAGGER, nil, xi.slot.MAIN)
+
+        -- Set TP to a specific value
+        player:setTP(1500)
```

Comment:
These sorts of comments are redundant, no?
`Set TP to a specific value` and `player:setTP(1500)` read as exactly the same thing, so this is redundant?

Reply:
Removed.